pkg/imgpkg/cmd/copy_repo_src.go

```diff
@@ -133,6 +160,11 @@ func (c CopyRepoSrc) CopyToRepo(repo string) (*ctlimgset.ProcessedImages, error)
 		return nil, fmt.Errorf("Tagging images: %s", err)
 	}
 
+	err = os.RemoveAll(tempDir)
+	if err != nil {
+		fmt.Println("Error cleaning up temporary directory:", err)
+	}
+
```

Comment:
Added a prior comment that I think it would be better to use than this part of the code. Because if anything fails in the meanwhile we will leave the folder back
```suggestion
```